modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostGISHStoreOnlineTest.java

```diff
@@ -124,6 +126,18 @@ public void testNullEntry() throws Exception {
         assertNull(object);
     }
 
+    @Test
```

Comment:
Hum ok... but the evaluation is still done in memory right?

Reply:
Yep. it's testing the MapGet filter function functionalities, not the HSTORE native support for extract an item with its key (which is still not implemented). If I have understood your comment, correctly.